docs/documentation/release_notes/topics/22_0_2.adoc

```diff
@@ -0,0 +1,4 @@
+= Improvements in LDAP and Kerberos integration
+
+It is possible to have multiple LDAP providers in the realm, which support Kerberos integration with same Kerberos realm. When LDAP provider is not able to find user, which was authenticated through
+Kerberos/SPNEGO, there will be attempt to fallback to next LDAP provider.
```

Comment:
I added some articles, and avoided the passive: 

```suggestion
Keycloak now supports multiple LDAP providers in a realm, which support Kerberos integration with the same Kerberos realm. When an LDAP provider is not able to find the user which was authenticated through
Kerberos/SPNEGO, Keycloak ties to fallback to the next LDAP provider.
```

Looking at the added tests, I wonder if we now support even more than is described here, like multiple Kerberos providers, but I'm not so much into this.  

I'm about to approve it anyway, and the docs could be further improved in a separate PR, as KC 22.0.2 might be just around the corner.